src/__experimental__/components/decimal/decimal.component.js

```diff
@@ -230,17 +154,6 @@ class Decimal extends React.Component {
   getSafePrecisionProp = () => {
     const { precision } = this.props;
 
```

Comment:
Do we want to remove these checks? It might be worth moving it into a custom prop type function